internal/cli/root/atlas/builder.go

```diff
@@ -113,7 +113,6 @@ func Builder() *cobra.Command {
 	rootCmd := &cobra.Command{
 		Version: version.Version,
 		Use:     atlas,
-		Aliases: []string{config.ToolName},
```

Comment:
[q] could this be a breaking change or does the alias not work in this case?

Reply:
It does nothing, it's a fake alias